Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# syntax=docker/dockerfile:1
+
+######################### BUILD STAGE 1 ############################
+# This stage builds the HarbourBridge binary using its source code.
+
+# Add the official Docker golang image as the base image for the build stage
+FROM golang:latest AS build
+
+# Set the default work directory as “/hbBin” in the container.
+WORKDIR /hbBin
+
+# Copy HarbourBrdige’s source code into the working directory.
+COPY . ./
+# Download HarbourBridge’s dependencies mentioned in the go.mod file.
+RUN go mod tidy 
+# Compile the source code and store the binary in the work directory /hbBin.
+RUN make build-static  
+
+######################### Release Stage ###########################
+# This stage consists of the GCloud SDK and the HabrbourBridge binary.
+
+# Add the Ubuntu base image from the Google Container Registry.
+FROM marketplace.gcr.io/google/ubuntu1804:latest
+# Set the default working directory.
+# This directory binds with the host directory that is to store the output files generated by HarbourBridge.
+WORKDIR /harbourBridge/harbour_bridge_output
+# Create directory to store the HarbourBridge binary. 
+RUN mkdir /harbourBridge/bin
+# Create directory to store the dump files. This directory binds with the host directory that contains the dump files of the source database.
+RUN mkdir /harbourBridge/sourceDump
+# Copy the HarbourBridge binary from the build stage's hbBin directory to the release stage's habourBridge/bin directory
+COPY --from=build /hbBin/harbourbridge /harbourBridge/bin
+
+# Add Python3 to support GCloud SDK.
+RUN apt update
+RUN apt install -y python3
```

Comment:
I think its best to mention a version everywhere possible

Reply:
Okay.

Reply:
Done.